apps/site/assets/ts/stop/components/DeparturesAndMap.tsx

```diff
@@ -41,6 +56,7 @@ const DeparturesAndMap = ({
   const { data: schedules } = useSchedulesByStop(stop.id);
   const predictions = usePredictionsChannel({ stopId: stop.id });
   const departureInfos = mergeIntoDepartureInfo(schedules || [], predictions);
+  const [realtimeAlerts, setRealtimeAlerts] = useState<Alert[]>([]);
```

Comment:
nitpick: `alertsForStopAndRoute` might be a clearer name compared to `realtimeAlerts`; to me "realtime" suggests some sort of system-driven "liveness" that isn't this...